cmd/snap-confine/udev-support.c

```diff
@@ -220,6 +220,10 @@ void setup_devices_cgroup(const char *security_tag, struct snappy_udev *udev_s)
 	const char *nvuvm_path = "/dev/nvidia-uvm";
 	const unsigned nvuvm_major = 247;
 	const unsigned nvuvm_minor = 0;
+	const char *nvidia_modeset_path = "/dev/nvidia-modeset";
+	const unsigned nvidia_modeset_major = 195;
```

Comment:
I think all of nvidia uses the same major number which we already define somewhere in the tree. It's fine to do this as a follow up though.